localstack/services/plugins.py

```diff
@@ -1,9 +1,13 @@
+import abc
 import json
 import logging
 import time
 from concurrent.futures import ThreadPoolExecutor
+from typing import Collection, Dict, Optional
 
 import requests
+from flask import Request
```

Comment:
do we plan to use anything else than flask for making HTTP requests inside localstack? if so, it maybe makes sense not to strongly couple to flask for now, and just leave the type of the `request` function parameters open for now.

Reply:
Ah, great catch. Replaced the import with `requests.models.Request` for now, as this is what we're using as the canonical request format in a few places already. We can think about changing it to something like `Union[FlaskRequest,RequestsRequest]` (or remove the type hint) in the future, if necessary.. 👍